backend/app/api/src/helpers/MemberCharger.ts

```diff
@@ -0,0 +1,56 @@
+import { SimpleError } from '@simonbackx/simple-errors';
+import { BalanceItem, Platform } from '@stamhoofd/models';
+import { BalanceItemType, MemberWithRegistrationsBlob } from '@stamhoofd/structures';
+import { BalanceItemService } from '../services/BalanceItemService';
+
+export class MemberCharger {
+    static async chargeFromPlatform(args: { membersToCharge: MemberWithRegistrationsBlob[]; price: number; amount?: number; description: string; dueAt: Date | null; createdAt: Date | null }) {
+        const platform = await Platform.getShared();
+
+        const chargeVia = platform.membershipOrganizationId;
+
+        if (!chargeVia) {
+            throw new SimpleError({
+                code: 'missing_membership_organization',
+                message: 'Missing membershipOrganizationId',
+                human: 'Er is geen lokale groep verantwoordelijk voor de aanrekening van aansluitingen geconfigureerd',
+            });
+        }
+
+        await MemberCharger.chargeMany({ chargingOrganizationId: chargeVia, ...args });
+    }
```

Comment:
Ik zou dit eigenlijk compleet uit de backend weglaten, en overlaten aan de frontend. Die kan dat ook en dan is het gedrag veel duidelijker. Je kan de endpoint dan gewoon volledig organization scoped maken zodat het niet meer optional is + de permission checks zijn meteen 100% accuraat.